apps/aecore/test/aecore_naming_test.exs

```diff
@@ -378,12 +391,12 @@ defmodule AecoreNamingTest do
     first_name_claim = NamingStateTree.get(naming_state_2, claim_hash_1)
 
     assert {:ok, first_name_claim.hash.value} == NameUtil.normalized_namehash("test.aet")
-    assert first_name_claim.owner.value == elem(Keys.keypair(:sign), 0)
+    assert first_name_claim.owner == elem(Keys.keypair(:sign), 0)
 
     assert first_name_claim.status == :claimed
-    assert first_name_claim.pointers == []
+    assert first_name_claim.pointers == "[]"
```

Comment:
Is it supposed to be like that? (a string=`[]` instead of a an empty list)?

Reply:
Yes, it is supposed to be like this because we are not using `Poison.encode([])`. We are just hard coding `"[]"`